apps/protocol-frontend/src/Routes.tsx

```diff
@@ -44,10 +45,11 @@ const RequireActiveUser = ({ children }: { children: JSX.Element }) => {
 };
 
 const RequireDaoUser = ({ children }: { children: JSX.Element }) => {
-  const { isUserDaoMember } = useUser();
   const { guildId } = useParams();
+  const { userData } = useUser();
+  const { data } = useUserGet({ userId: userData?.id });
```

Comment:
Great idea having the hook call within the RequireDaoUser so that we're only making this query when a user is navigating to a Route wrapped with this. One worry I had was that we'd be making this call on every route change, but the wrapping avoids that.